src/chttpd_db.erl

```diff
@@ -93,7 +93,7 @@ handle_changes_req1(#httpd{}=Req, Db) ->
             couch_stats:decrement_counter([couchdb, httpd, clients_requesting_changes])
         end;
     _ ->
-        Msg = <<"Supported `feed` types: normal, continuous, longpoll, eventsource">>,
+        Msg = <<"Supported `feed` types: normal, continuous/stream, longpoll, eventsource">>,
```

Comment:
`Supported `feed` types: normal, continuous, stream, longpoll, eventsource` e.g. without making difference between continuous and stream since both are the valid options. All the questions "why" will be covered by the docs.